src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-style: normal;
+  font-weight: 800;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc") format("truetype");
+  font-style: normal;
+  font-weight: 600;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  font-family: Avenir-Book, Avenir-Heavy, Avenir, Arial, Helvetica, sans-serif;
```

Comment:
move `font-family` to `html` selector, and use `font-family: inherit;` for other elements